### PR TITLE
feat: add component name to overlay

### PR DESCRIPTION
**Pull request by raunofreiberg**
To get the ball rolling on #4.

Also, modified colors a bit to match the current theme 💅

image

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -16,6 +16,20 @@ function validateNode(node: ElementContext): Promise<Result[]> {
...
 }
 
+// Copied from:
+// https://stackoverflow.com/questions/29321742/react-getting-a-component-from-a-dom-element-for-debugging
+// ¯\_(ツ)_/¯
+function getComponentFromNode(node: HTMLElement): string {
+  const [component] = Object.keys(node)
+    .filter(key => key.startsWith('__reactInternalInstance$'))
+    .map((key: string) => {
+      const fiberNode = (node as any)[key];
```
Comment (raunofreiberg) [inline src/index.tsx:26]: I can't figure the types out here so I just told TS to shut up, for now.

Comment (chancestrickland) [reply src/index.tsx:26]: Don't be afraid to tell TS to shut it's annoying whining from time to time!

**Comment by chancestrickland** [comment]
Looks good to me!

File: src/index.tsx (hunk 1 above)
Comment (raunofreiberg) [reply src/index.tsx:26]: How else would you expect it to learn some manners 😄

**Pull request merged**